src/app/submission/sections/accesses/section-accesses.service.ts

```diff
@@ -0,0 +1,42 @@
+import { Injectable } from '@angular/core';
+
+import { Observable } from 'rxjs';
+import { distinctUntilChanged, filter } from 'rxjs/operators';
+import { Store } from '@ngrx/store';
+
+import { SubmissionState } from '../../submission.reducers';
+import { isUndefined } from 'util';
+import { submissionSectionDataFromIdSelector } from '../../selectors';
+
+
+/**
+ * A service that provides methods to handle submission's bitstream state.
```

Comment:
Again, not sure this is actually specific to bitstreams.  Isn't this related to the Access section/panel?